src/App.jsx

```diff
@@ -14,33 +15,116 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [sortedAlph, setSortedAlph] = useState(false);
+  const [sortedByLength, setSortedByLength] = useState(false);
```

Comment:
```suggestion
  const [sortType, setSortType] = useState(SORT_TYPES.none);
```
If you have more sort types you have to add more states. You can store a sort type instead